<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="forsa-vm31" failures="1" tests="7" name="TopZinsPortal.TZPCheckEMailsGG" time="1.875" errors="0" timestamp="2021-04-30T17:00:16 CEST" skipped="0">
  <testcase classname="TopZinsPortal.TZPCheckEMailsGG" name="TZPCheckEmailGGTest" time="0.006"/>
  <system-out/>
  <testcase classname="TopZinsPortal.TZPCheckEMailsGG" name="TZPCheckEmailGGTest" time="0.001"/>
  <system-out/>
  <testcase classname="TopZinsPortal.TZPCheckEMailsGG" name="TZPCheckEmailGGTest" time="1.860">
    <failure message="The following asserts failed:
	expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at TopZinsPortal.TZPCheckEMailsGG.TZPCheckEmailGGTest(TZPCheckEMailsGG.java:107)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TZPCheckEmailGGTest -->
  <system-out/>
  <testcase classname="TopZinsPortal.TZPCheckEMailsGG" name="TZPCheckEmailGGTest" time="0.002"/>
  <system-out/>
  <testcase classname="TopZinsPortal.TZPCheckEMailsGG" name="TZPCheckEmailGGTest" time="0.003"/>
  <system-out/>
  <testcase classname="TopZinsPortal.TZPCheckEMailsGG" name="TZPCheckEmailGGTest" time="0.002"/>
  <system-out/>
  <testcase classname="TopZinsPortal.TZPCheckEMailsGG" name="TZPCheckEmailGGTest" time="0.001"/>
  <system-out/>
</testsuite> <!-- TopZinsPortal.TZPCheckEMailsGG -->
